### PR TITLE
UI improvements for classic theme

### LINKED ISSUE #12910: [CLASSIC THEME] UI improvements
List of several UI improvements on the classic theme : 

**General :**

- Change the grey background color to #F6F6F6 on every page

- Add a fade-in / fade-out animation on menu hover 

**Product page :** 

- Change the "zoom" hover on large product image:
Capture d’écran 2019-03-12 à 12 09 47

Specs : Filter #00000 60% opacity ; icon size: 50x50px ; replace the current icon by metariel icon "Search"

### PR DESCRIPTION
**Pull request by YeLnatSs**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | UI improvements on various pages
| Type?         | improvement
| Category?     | FO
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | Fixes #12910 
| How to test?  | N/a

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/13383)

### CODE REVIEW

File: themes/classic/_dev/css/partials/_variables.scss [hunk 1]
```diff
@@ -5,6 +5,7 @@ $gray-light: #F1F1F1;
 $gray-light-second: #DBDBDB;
 $gray-lighter: #f6f6f6;
 $gray-lighter-second: #F7F7F7;
+$black: #000;
```
Comment (Quetzacoalt91) [inline themes/classic/_dev/css/partials/_variables.scss]: Is this really needed? If we were dealing with complex color, I wouldn't say anything, but this case is simple enough for not over-engineering it.

Comment (YeLnatSs) [reply themes/classic/_dev/css/partials/_variables.scss]: No problem, I can remove it from variables files if needed

**Review by Quetzacoalt91 — DISMISSED** [review]

**Comment by marionf** [comment]
@YeLnatSs 

Opacity on the hover on product page is 70% instead of 60%

capture d'écran_1615

**Comment by YeLnatSs** [comment]
@marionf I changed to 60%

**Comment by PierreRambaud** [comment]
PR rebased, waiting for Ci

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @YeLnatSs

**Pull request merged**